corehq/apps/hqwebapp/migrations/0006_create_user_access_log.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.16 on 2021-02-09 15:56
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hqwebapp', '0005_delete_apikeysettings'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UserAgent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('value', models.CharField(db_index=True, max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='UserAccessLog',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('user_id', models.CharField(db_index=True, max_length=255)),
+                ('action', models.CharField(choices=[('login', 'Login'), ('logout', 'Logout'),
+                    ('failure', 'Login Failure')], max_length=20)),
+                ('ip', models.GenericIPAddressField(blank=True, null=True)),
+                ('path', models.CharField(blank=True, max_length=255)),
+                ('timestamp', models.DateTimeField(default=django.utils.timezone.now)),
```

Comment:
This default should be updated if the suggestion on the model field is accepted.